ts-client/src/dlmm/index.ts

```diff
@@ -1095,6 +1112,10 @@ export class DLMM {
     );
     const program = new Program(IDL, LBCLMM_PROGRAM_IDS[opt.cluster], provider);
 
+    if (!this.isPoolExisted(connection, tokenX, tokenY, binStep, baseFactor)) {
```

Comment:
why not isPoolExisted return Error Pool already exists?

Reply:
ohh sorry.